components/atom/tag/src/index.js

```diff
@@ -5,12 +5,25 @@ import classnames from 'classnames'
 import ActionableTag from './Actionable'
 import StandardTag from './Standard'
 
+const SIZES = {
+  MEDIUM: 'medium',
+  SMALL: 'small'
+}
+
 class AtomTag extends Component {
   static MAX_LABEL_LENGTH = 100
 
+  /**
+   * @return {string}
+   */
   get _classNames() {
-    const {className, icon} = this.props
-    return classnames('sui-AtomTag', className, icon && 'sui-AtomTag-hasIcon')
+    const {className, icon, size} = this.props
+    return classnames(
+      'sui-AtomTag',
+      `sui-AtomTag-${size}`,
+      className,
```

Comment:
As you are working on this component, could you please remove the className prop completely as we don't want to make components "styleable" from outside?

Reply:
Aren't we breaking the API removing className?

Reply:
Being strict we should create a major, but it looks like there is nobody using it

https://www.npmjs.com/package/@schibstedspain/sui-atom-tag
https://github.schibsted.io/search?q=sui-atom-tag&type=Code&utf8=✓